File: src/main/java/net/igneo/icv/entity/boots/soulEmber/SoulEmberRenderer.java
package net.igneo.icv.entity.boots.soulEmber;

import net.minecraft.client.renderer.entity.EntityRendererProvider;
import software.bernie.geckolib.renderer.GeoEntityRenderer;

public class SoulEmberRenderer extends GeoEntityRenderer<SoulEmberEntity> {
    public SoulEmberRenderer(EntityRendererProvider.Context renderManager) {
        super(renderManager, new SoulEmberModel());
    }
}

File: src/main/java/net/igneo/icv/entity/boots/soulEmber/SoulEmberRenderer.kt
package net.igneo.icv.entity.boots.soulEmber

import net.minecraft.client.renderer.entity.EntityRendererProvider
import software.bernie.geckolib.renderer.GeoEntityRenderer

class SoulEmberRenderer(renderManager: EntityRendererProvider.Context) :
    GeoEntityRenderer<SoulEmberEntity?>(renderManager, SoulEmberModel())
